packages/grafana-ui/src/components/Forms/InlineLabel.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { FunctionComponent, HTMLProps, ReactNode } from 'react';
+import { GrafanaTheme } from '@grafana/data';
+import { css, cx } from 'emotion';
+import { Tooltip, PopoverContent } from '../Tooltip/Tooltip';
+import { Icon } from '../Icon/Icon';
+import { useTheme } from '../../themes';
+
+export interface Props extends Omit<HTMLProps<HTMLLabelElement>, 'children' | 'className' | 'css'> {
```

Comment:
Think this should extend [Label props](https://github.com/grafana/grafana/blob/416111c5f6afb32c4036724d7f87fa7a1102055e/packages/grafana-ui/src/components/Forms/Label.tsx#L8) to keep the interfaces in sync. WDYT?

Reply:
Good point! Haven' thought about that.